jest.config.js

```diff
@@ -8,5 +8,6 @@ module.exports = {
     "^.+\\.tsx?$": "ts-jest",
   },
   moduleFileExtensions: ["js", "ts", "json", "node"],
-  collectCoverage: true,
+  // Uncomment this if we want to use coverage
+  // collectCoverage: true,
```

Comment:
We should use this as we add more tests.

Reply:
That will be addressed in this ticket: https://trello.com/c/A578aCfI/2276-plan-backend-tests-of-app